modules/gcp_project/variables.tf

```
@@ -0,0 +1,43 @@
+variable "project_id" {
```

Make the name of every resource configurable instead of generating non-deterministic names.
Same as the other submodules.
```suggestion
variable "iam_name_prefix" {
  description = "The prefix given to all GCP IAM resource names."
  type        = string
  default     = "IllumioCloudIntegration"
  validation {
    condition     = length(var.iam_name_prefix) > 0
    error_message = "The iam_name_prefix value must not be empty."
  }
}

variable "project_id" {
```